src/backend/deploy.js

```diff
@@ -1,64 +1,67 @@
+/* eslint-disable no-empty-label */
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rule 'space-return-throw-case' was removed and replaced by: keyword-spacing (space-return-throw-case)](https://app.codacy.com/app/triplez/open-event-webapp/pullRequest?prid=3226522)